### DISCUSSION TITLE
RELATED synonym with typedef syngo_official label

### DISCUSSION THREAD

**Discussion by stephanielipow** [geneontology/helpdesk#444] [General]
[Term]

Very minor anomaly/outlier run across while processing GO for the UMLS:  in this record there is a RELATED synonym with a typedef syngo_official_label.  All others with this typedef are
EXACT synonyms:

 

> id: GO:0099565
> name: chemical synaptic transmission, postsynaptic
> namespace: biological_process
> def: "The part of synaptic transmission occurring in the post-synapse: a signal transduction pathway consisting of neurotransmitter receptor activation and its effects on postsynaptic membrane potential and the ionic composition of the postsynaptic cytosol." [GOC:dos]
> subset: goslim_synapse
> **synonym: "postsynaptic process involved in chemical synaptic transmission" RELATED syngo_official_label []**
> is_a: GO:0050877 ! nervous system process
> is_a: GO:1905114 ! cell surface receptor signaling pathway involved in cell-cell signaling
> intersection_of: GO:0007166 ! cell surface receptor signaling pathway
> intersection_of: occurs_in GO:0098794 ! postsynapse
> intersection_of: part_of GO:0007268 ! chemical synaptic transmission
> relationship: occurs_in GO:0098794 ! postsynapse
> relationship: part_of GO:0007268 ! chemical synaptic transmission

**Comment by balhoff** [comment]
I'm going to move this to the ontology issues so the right folks will take a look.

**Comment by balhoff** [comment]
https://github.com/geneontology/go-ontology/issues/25649
